device/isis/v6routerange.yaml

```diff
@@ -0,0 +1,46 @@
+components:
+  schemas:
+    Router.IsisV6Route:
+      x-include: 
+      - './routerange.yaml#/components/schemas/Router.IsisRoute'
+      description: >-
+        Emulated ISIS v6 route
+      type: object
+      properties:
+        addresses:
+          description: >-
+            A list of symmetrical or asymmetrical route addresses
+          type: array
+          items:
+            $ref: '#/components/schemas/Router.Isis.V6RouteAddress'
+      
+    Router.Isis.V6RouteAddress:
+      description: >-
+        A container for Isis Ipv6 route addressses
+      type: object
+      properties:
+        address:
+          description: >-
+            The starting address of the network
+          type: string
+          format: ipv6
+          default: ::0
```

Comment:
is this a property we expect user to change 99% of time?  If yes, make it required/remove the default.